druid-shell/src/application.rs

```diff
@@ -23,6 +23,11 @@ use crate::platform::application as platform;
 pub struct Application;
 
 impl Application {
+    /// Initialize the app. At the moment, this is mostly needed for hi-dpi.
```

Comment:
Longer term I think we might want to look at getting rid of this, as it's a small source of noise, but I think appropriate to consider this separately from this refactor.